templates/ui-frameworks/vue/web-app/ui/src/clientProvider.ts.hbs

```diff
@@ -0,0 +1,76 @@
+import {
+  type AppClient,
+  AppWebsocket,
+  HolochainError,
+} from "@holochain/client";
+{{#if holo_enabled}}
```

Comment:
Still waiting for confirmation from holo on whether we should continue maintaining this, leaving it here for now though

Reply:
Who are you waiting for for an answer?

Reply:
From Alastair, followed up earlier this week, no update yet